vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -411,13 +367,20 @@ type
 
     let optionsAssociation = ConditionalWeakTable<IWorkspaceProjectContext, string[]>()
 
-    member private this.OnProjectAdded(projectId:ProjectId, _newSolution:Solution) = projectInfoManager.UpdateProjectInfoWithProjectId(projectId, "OnProjectAdded")
+    member private this.OnProjectAdded(projectId:ProjectId, _newSolution:Solution)    = projectInfoManager.UpdateProjectInfoWithProjectId(projectId, "OnProjectAdded")
```

Comment:
If ``_newSolution`` is unused then why not remove it?

Reply:
I suspect that we will be using eventually.  It's just not necessary right now.